README.md

```diff
@@ -188,6 +188,69 @@ pip install 'apache-airflow[postgres,google]==3.0.0' \
 For information on installing provider distributions, check
 [providers](http://airflow.apache.org/docs/apache-airflow-providers/index.html).
 
+3. Setting up Apache Airflow on Mac M1 or above (Apple Silicon) users
+
```

Comment:
This one looks like quite a bit chat-gpt generated and it's not entirely what I woudl suggest.

First question: What's the difference vs. Mac M1 and Linux here ?